org.eclipse.lsp4xml/src/main/java/org/eclipse/lsp4xml/commons/TextDocuments.java

```diff
@@ -26,21 +26,13 @@
  */
 public class TextDocuments implements ITextDocumentFactory {
 
-	private boolean incremental;
+	private final Boolean incremental;

```

Comment:
why do you need a Boolean?

Reply:
was used as a safety safeguard for a previous implementation, removed now.